tests/unit/copying_manager_test.py

```diff
@@ -1049,6 +1040,8 @@ def test_pipelined_requests(self):
 
         responder_callback("success")
 
+    # TODO: confirm that this test only makes sense in the context of the single worker, and then remove.
+    @skipIf(True, "This test is moved to worker tests.")
```

Comment:
I think it makes sense that you just have to test it in the single worker case.  We still need the test though.